packages/Toast/src/Toast.styles.js

```diff
@@ -86,7 +88,7 @@ export const Toast = styled.div(
     font-weight: normal;
     margin-bottom: ${stylers.spacer(2)};
     min-height: ${stylers.spacer(3)};
-    padding: ${tokens.spaceLg};
+    padding: 11px ${tokens.spaceLg};
```

Comment:
With the negative margin applied to the close button now, this can revert back to 12px all around.  Currently with 11px it's just a _little_ off-center (I wasn't sure if it was just me or not, but got a designer to confirm it's just a little off).

Reply:
@mikrotron I can't add it to 12px as there is a height change request for toast. I need to reduce it from 50px to 48px. The negative margin is there to reduce the button height(32px) to the same height as the text next to it so it won't push the container when it is one line. I add the negative margin to raise the button icon up a bit. Hope it helps.

@jonathanjakimon 

![Screen Shot 2021-11-16 at 10 54 46 AM](https://user-images.githubusercontent.com/92405042/142047824-9c44804d-ced0-487a-a0bb-9ae2d29f6853.png)

Reply:
Oh, I see 🤔 

Can you compare a one line and two line `<Toast>` with `padding: 11px ${tokens.spaceLg};` with another pair of `<Toasts>` that have `padding: ${tokens.spaceLg} ${tokens.spaceLg} 10px ${tokens.spaceLg};`?  Let's let Jonathan or Carol decide which version they prefer and go with that.

Reply:
Sure thing.  
Here is 12px padding top and bottom makes the toasts to 50px when single line.
<img width="1569" alt="Screen Shot 2021-11-16 at 12 48 56 PM" src="https://user-images.githubusercontent.com/92405042/142063565-08041c4c-bf38-4f53-bb75-f7559b91258f.png">

Here is 11px padding top and bottom which makes the toasts 48px in heights when single line.
<img width="1529" alt="Screen Shot 2021-11-16 at 12 48 03 PM" src="https://user-images.githubusercontent.com/92405042/142063719-6608d7ae-f1d4-40e7-82a1-ed9fe003a222.png">

@jonathanjakimon and @carolchaan Can you let me know which one works best for you.

Reply:
After using the pixel ruler I can say that 12px has better alignments in general :)

Reply:
Cool thank you @jonathanjakimon let me push the change :)